extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -27,14 +27,14 @@ class AmpStickyAd extends AMP.BaseElement {
   constructor(element) {
     super(element);
 
+    /** @const @private {!../../../src/service/vsync-impl.Vsync} */
+    this.vsync_ = this.getVsync();
```

Comment:
thanks, yeah for the most part i have made them nullable and moved all the calls to buildCallback